_sass/_layout.scss

```diff
@@ -243,3 +243,34 @@
 .pagination {
   text-align: center;
 }
+
+// Social Media Biography
+
+.bio-social-media {
+  ul {
+    margin-top: -$base-margin-bottom;
+    margin-bottom: $base-margin-bottom;
+    padding: 0;
+    width: 100%;
+    list-style: none;
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  li {
+    margin-right: $spacing-unit / 2;
+    margin-top: 0;
+    padding: 5px 0;
+    float: left;
+    width: 30%;
```

Comment:
I'm finding this to be a bit strange to see this items lined on the same edge. I think it's better to have them flow against each other without a fixed width.